packages/api/src/command/medical/document/document-query-storage-info.ts

```diff
@@ -100,7 +100,7 @@ export async function getS3Info(
     const msg = `Errors getting info from S3`;
     const extra = { errors, patientId: patient.id };
     log(msg, extra);
-    capture.message(msg, { extra });
+    capture.error(msg, { extra });
```

Comment:
True... OTOH, we're using a message, which already points us to this code. Can do if I send another commit, otherwise I'd leave as is to avoid changing more stuff.